### ISSUE TITLE
Connection Issue with Neovim Instance on /tmp/nvimsocket in Obsidian-Neovim Companion Plugin

### ISSUE THREAD

**Issue by SureshPradhana** [B4rc1/obsidian-companion.nvim#1]
I encountered a problem when trying to activate the obsidian-neovim-companion plugin in Obsidian. Even though I installed the obsidian-companion.nvim plugin for Neovim, I received a warning message saying:
```
Warning: Unable to connect to any Neovim instance on /tmp/nvimsocket.
```

**Comment by B4rc1** [comment]
did you start neovim with `nvim --listen /tmp/nvimsocket`?

**Comment by SureshPradhana** [comment]
Yes, I started the Neovim with the command `nvim --listen /tmp/nvimsocket`
To check the number of connections, I used the command `ls -l /tmp/nvimsocket` , which showed there are 0 connections.

**Comment by B4rc1** [comment]
Starting neovim with `nvim --listen /tmp/nvimsocket`, THEN opening obsidian and enabling the plugin works on my machine with
```
$ nvim --version
NVIM v0.9.1
Build type: Release
LuaJIT 2.1.0-beta3

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/nix/store/mmp203bf0m9cz39di4pj1xq456ffjwbf-neovim-unwrapped-0.9.1/share/nvim
"

Run :checkhealth for more info
```
and Obsidian version v1.3.5.

Connection during runtime is not implemented and support/updates for this plugin are unlikely due to the excellent [Latex Suit](https://github.com/artisticat1/obsidian-latex-suite) which implements everything I wanted this plugin for, namely snippets to fast and painlessly write latex.
Obsidian vim emulation works well enough for my purposes.
